### DISCUSSION TITLE
Default registry for helm-values manager

### DISCUSSION THREAD

**Discussion by lictw** [renovatebot/renovate#30209] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

37

### Please tell us more about your question or problem

I have additional `values` files with overrides and I want to have only tag/version there, without additional copy of registry and repository. Can helm-values (or other one, but not custom) manager work with files where only `image.version` specified for docker images?

I understant and can do it all with custom manager, you don't have to explain it. I am interested exactly in standard managers.

**Comment by RahulGautamSingh** [comment]
It should not be possible. Can you show an example of the `values.yaml` file you want to work with?
